platform/ios/src/MGLMapView.h

```diff
@@ -597,7 +597,8 @@ MGL_EXPORT IB_DESIGNABLE
  * If the value of this property is smaller than that of the
  * minimumZoomLevel property, the behavior is undefined.
  *
- * The default maximumZoomLevel is 20.
+ * The default maximumZoomLevel is 22. The upper bound for this property
```

Comment:
@1ec5 - In this case its referring to itself, so this shouldn't need any linking.